probes/hasDangerousWorkflowScriptInjection/patch/testdata/twoInjectionsSameStep.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2021 OpenSSF Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+on:
+  fork:
+  issue_comment:
+    types: [created, edited]
+
+jobs:
+  solution-to-all-repo-problems:
+    steps:
+      - name: where things are done
+        run: |
+          echo "${{ github.event.issue_comment.comment }}"
+          mkdir "${{ github.event.fork.forkee.name }}"
```

Comment:
`fork.forkee.name` isn't a dangerous input. It is user-controlled, but the rules for repo names make it impossible to craft a malicious name (no spaces, no slashes, etc).